scrapy/resolver.py

```diff
@@ -4,15 +4,20 @@
 from scrapy.utils.datatypes import LocalCache
 
 # TODO: cache misses
-# TODO: make cache size a setting
 
 dnscache = LocalCache(10000)
```

Comment:
Shouldn't this limit change with `DNSCACHE_SIZE`? Or is this the upper hard limit.